ern-api-impl-gen/src/generators/android/ApiImplMavenGenerator.js

```diff
@@ -61,36 +63,48 @@ export default class ApiImplMavenGenerator implements ApiImplGeneratable {
       shell.cd(ROOT_DIR)
 
       const outputDirectory = path.join(paths.outDirectory, `android`)
```

Comment:
Not a change from this PR but could you please replace backticks with single quotes here as part of this PR.